docker-compose.yml

```diff
@@ -15,6 +15,8 @@ services:
 
   grpc-web-proxy:
     build: ./grpc-web-proxy/
+    environment:
```

Comment:
I think instead of environment variables, the build args should be passed into the docker build context: https://docs.docker.com/compose/compose-file/ (search for build-args)
I have it implemented like this: 
```
build:
      context: ./grpc-web-proxy/
      args:
        - GRPC_SERVER=lieferemma-backend
```

Reply:
I assumed it would pick the env vars. Updated it. Thanks!